features/users/friendship.service.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+var debugEnabled = require("../../config").debug;
+var log = require("color-logs")(debugEnabled, debugEnabled, __filename);
+var Promise = require('promise');
```

Comment:
Any reason you choose this implementation of promise instead of native promise of nodejs?

Reply:
pure stupidity of my part. I was doing some tests and forgot to remove this `require`.